### PR TITLE
Automated DevTools release process

### PR DESCRIPTION
**Pull request by bvaughn**
- [x] Automate Git commit
- [x] Automate build and `yarn build` and `npm publish` steps
- [x] Add publish instructions for extension stores

https://user-images.githubusercontent.com/29597/130663304-85272de1-2b86-4dc5-90ac-ed44cf26cad7.mp4

### CODE REVIEW

File: packages/react-devtools-extensions/chrome/manifest.json [hunk 1]
```diff
@@ -4,48 +4,49 @@
   "description": "Adds React debugging tools to the Chrome Developer Tools.",
```
Comment (jstejada) [inline packages/react-devtools-extensions/chrome/manifest.json:4]: is this commit relevant to the script in some way?

File: packages/react-devtools/CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,6 @@
...
 
-<details>
-  <summary>
-    Changes that have landed in main but are not yet released.
-    Click to see more.
-  </summary>
-
-  <!-- Upcoming changes go here -->
-</details>
+<!-- RELEASE_SCRIPT_TOKEN -->
```
Comment (jstejada) [inline packages/react-devtools/CHANGELOG.md:3]: ooh interesting

File: scripts/devtools/release.js [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+    console.log(chalk.dim('  Updating manifest JSON: ' + partialPath));
+
+    if (!DRY_RUN) {
+      writeJsonSync(path, json, {spaces: 2});
+    }
+  });
+}
+
+function updatePackageVersions(previousVersion, nextVersion) {
```
Comment (jstejada) [inline scripts/devtools/release.js:217]: ❤️

**Review by jstejada — APPROVED** [review]
looks pretty great so far! pre-emptive approval. one suggestion is that you might want to change the title of the PR?

File: packages/react-devtools-extensions/chrome/manifest.json (hunk 1 above)
Comment (bvaughn) [reply packages/react-devtools-extensions/chrome/manifest.json:4]: Yes. I updated the `manifest.json` files in their own commit to match the format generated by `JSON.stringify(json, null, 2)` so the version update script would _only_ affect version numbers (and no other formatting chagnes).

**Comment by bvaughn** [comment]
> looks pretty great so far! pre-emptive approval. one suggestion is that you might want to change the title of the PR?

Thanks!

Yeah, GitHub defaults a PR title to the first commit 😂 Fixed.

**Review by jstejada — APPROVED** [review]
awesome, thanks!!

**Review by lunaruan — APPROVED** [review]
